INPTP_AppForFixing/MainForm.cs

```diff
@@ -44,18 +53,37 @@ public void OnEmployeeChange()
             if (CheckSelectedBoss())
             {
                 Boss temp = listBoxOfBosses.SelectedItem as Boss;
-                if (temp.GetEmployees().Count > 0)
+                if (temp.GetEmployees().Count >= 0)
                 {
                     listBoxEmpl.DataSource = new List<Employee>(temp.GetEmployees());
-                }                
-            }            
+                }
+            }
+        }
+
+        public Boss GetSelectedBoss() => (Boss)listBoxOfBosses.SelectedItem;
+
+        public Employee GetSelectedEmployee() => (Employee)listBoxEmpl.SelectedItem;
+
+
+        public int getNextBossId()
```

Comment:
Boss is in fact an employee. Why bosses have separate id creation? IDs between bosses and employees should be same and unique.

Reply:
fixed